public/app/features/logs/utils.ts

```diff
@@ -273,3 +273,27 @@ export const getLogsVolumeDataSourceInfo = (dataFrames: DataFrame[]): { name: st
 export const isLogsVolumeLimited = (dataFrames: DataFrame[]) => {
   return dataFrames[0]?.meta?.custom?.logsVolumeType === LogsVolumeType.Limited;
 };
+
+export const copyText = async (text: string, buttonRef: React.MutableRefObject<Element | null>) => {
+  if (navigator.clipboard && window.isSecureContext) {
```

Comment:
I was wondering if we need the fallback? 
`writeText` seems to be supported by every browser since latest early 2020: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText